### PR TITLE
Fix record support

### PR DESCRIPTION
**Pull request by hesham**

### CODE REVIEW

**Comment by hesham** [comment]
[ch3071]

File: lib/fortify/base.rb [hunk 1]
```diff
@@ -55,11 +55,11 @@ def cannot(action, *fields)
       access_list.delete(action) if access_list[action].empty?
     end
 
-    def initialize(record)
+    def initialize(object)
       raise InvalidUserError.new("Fortify user not set") if !Fortify.user
 
       @user = Fortify.user
-      @record = record
+      @record = object.is_a?(Class) ? object.new : object
```
Comment (hesham) [inline lib/fortify/base.rb]: Not sure if there's a better way to do this. Any ideas?

Context: https://app.clubhouse.io/updater/story/3071/fortify-policy-record-support-broken

Comment (justin-rhoades) [reply lib/fortify/base.rb]: I think this works, the only other thing would be `responds_to? :new`

Comment (hesham) [reply lib/fortify/base.rb]: @justin-rhoades i like `responds_to? :new` more.. thanks!

**Pull request merged**